src/main/java/frc/robot/competition/CompetitionRobotContainer.java

```diff
@@ -421,6 +423,10 @@ private void configureBindings() {
 
     m_testController.x().whileTrue(m_feedback.rainbows());
     m_testController.b().whileTrue(m_feedback.setColor(Color.kBlue));
+    m_testController
+        .y()
+        .whileTrue(
+            new AlignToNote(m_chassis, () -> new ChassisSpeeds(2, 0, 0)).alongWith(pickUpNote()));
```

Comment:
We probably want this to also deadline with so that if we test it on a practice field the robot doesn't keep driving